src/core/usecases/get-lessons-by-chapter.use-case.ts

```diff
@@ -1,26 +1,26 @@
 import { UseCase } from '../base/use-case';
-import { Unit } from '../domain/model/Unit';
-import { UnitRepository } from '../domain/repository/unit.repository';
+import { Lesson } from '../domain/model/Lesson';
+import { LessonRepository } from '../domain/repository/lesson.repository';
 import { User } from '../domain/model/User';
 import { UserType } from '../domain/type/UserType';
 import { UserNotAllowedError } from '../domain/error/UserNotAllowedError';
 
-export type getUnitsByChapterIdCommand = {
+export type getLessonsByChapterIdCommand = {
```

Comment:
(Refactor pas accept change pour éviter les problèmes) ;)
```suggestion
export type GetLessonsByChapterIdCommand = {
```